### DISCUSSION TITLE
how to highlight selected text occurrences

### DISCUSSION THREAD

**Discussion by icyhoty2k** [sumatrapdfreader/sumatrapdf#2938] [Q&A]
im reading a pdf and need to select a word and the see all occurrences in the page of that word can this be done somehow?

**Comment by GitHubRulesOK** [comment]
Not in SumatraPDF since that is a very complex text suite worth a bigger editor type application, so it cannot be done in chrome/edge without acrobat reader extension. there are many standalone programs such as pdfgrep https://stackoverflow.com/a/72973168/10802527 but if you want graphical I suggest lighter weight dngrep here on github 6MB as installer http://dngrep.github.io/ or better 58MB befor install [agent ransack](https://www.mythicsoft.com/agentransack/)
![image](https://user-images.githubusercontent.com/12754913/188632468-72e47122-f815-49e3-b226-1c47c3be0efa.png)
